packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -201,16 +187,17 @@ class FlutterDriver {
     if (Platform.isFuchsia && isolateNumber == null) {
       fuchsiaModuleTarget ??= Platform.environment['FUCHSIA_MODULE_TARGET'];
       if (fuchsiaModuleTarget == null) {
-        throw DriverError('No Fuchsia module target has been specified.\n'
-            'Please make sure to specify the FUCHSIA_MODULE_TARGET\n'
-            'environment variable.');
+        throw DriverError(
+          'No Fuchsia module target has been specified.\n'
+          'Please make sure to specify the FUCHSIA_MODULE_TARGET '
+          'environment variable.'
+        );
       }
       final fuchsia.FuchsiaRemoteConnection fuchsiaConnection =
           await FuchsiaCompat.connect();
       final List<fuchsia.IsolateRef> refs =
           await fuchsiaConnection.getMainIsolatesByPattern(fuchsiaModuleTarget);
       final fuchsia.IsolateRef ref = refs.first;
-      await Future<void>.delayed(_fuchsiaDriveDelay(timeoutMultiplier));
```

Comment:
What was the point of this delay?

Reply:
I spoke to the person who added it, and they weren't sure but seemed to think it was prophylactic.